src/main/resources/liquibase/202012141133-enforce-display-name-not-null.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <property name="table.name" value="hosts"/>
+    <property name="column.name" value="display_name"/>
+
+    <changeSet id="202012141133" author="lburnett">
+        <comment>
+            Add not null constraint to ${table.name}.${column.name} and set existing null values to empty
+            string
+        </comment>
+        <addNotNullConstraint tableName="${table.name}" columnName="${column.name}" defaultNullValue=""/>
+    </changeSet>
```

Comment:
Can you please inline table.name and column.name?

Reply:
all set